lib/package.gi

```diff
@@ -2402,7 +2404,7 @@ InstallGlobalFunction( ValidatePackageInfo, function( info )
        and IsBound( record.Persons ) then
       for subrec in record.Persons do
         TestMandat( subrec, "LastName", IsString, "a string" );
-        TestMandat( subrec, "FirstNames", IsString, "a string" );
+        TestOption( subrec, "FirstNames", IsString, "a string" );
```

Comment:
If we make it optional here we should also adjust `doc/ref/gappkg.xml` to say it is (now) optional. (Indeed besides moving it from the list of required components to optional ones, I'd add some text to the effect "since GAP 4.14 this is optional" or so